ts/sagas/startup/__tests__/checkProfileEmailSaga.test.tsx

```diff
@@ -15,7 +15,7 @@ import { appReducer } from "../../../store/reducers";
 import { renderScreenFakeNavRedux } from "../../../utils/testWrapper";
 import { checkAcknowledgedEmailSaga } from "../checkAcknowledgedEmailSaga";
 
-describe("checkAcceptedTosSaga", () => {
+describe("checkAcknowledgedEmailSaga", () => {
```

Comment:
Does this relate to the PR? 🤔

Reply:
No, it is unrelated and a clear violation of our PR policies. Nonetheless, unwilling to miss the opportunity of flutter two hearts with one look, after carefully weighing both the magnitude and impact of such unrelated change, I have decided to include it anyway (in this PR)

Reply:
Ah. Got it! Right then, let's leave it here.